pages/linux/brew.md

```diff
@@ -0,0 +1,27 @@
+# brew
+
+> The Homebrew package manager for Linux.
+
+- Search formula:
+
+`brew search {{text}}`
+
+- Install formula:
+
+`brew install {{formula}}`
+
+- List installed formulae [with matching name]:
+
+`brew list {{[text]}}`
```

Comment:
This page is identical to osx/brew.md, except for the description. I'd rather make any changes in another PR. As for this particular notation, I suppose it means that the parameter is optional; without, the full list would be printed. But I agree it isn't super-intuitive. Let's change it in a separate PR.